dataland-api-key-manager/src/test/kotlin/org/datalandapikeymanager/services/ApiKeyManagerTest.kt

```diff
@@ -13,9 +13,7 @@ import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.boot.jdbc.EmbeddedDatabaseConnection
 import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase
 import org.springframework.boot.test.context.SpringBootTest
-import org.springframework.context.annotation.ComponentScan
 
-@ComponentScan(basePackages = ["org.dataland"])
```

Comment:
Why is this no longer necessary?

Reply:
It never was. This was added to scan for all available components. But the `@SpringBootTest(...)` annotation basically did the same, i.e. it just created the components we had for the application. For the application I excluded the email builder now. `@ComponentScan` would have included it still. But because it was unnecessary anyways, I removed it.

Reply:
ok